docker/Dockerfile

```diff
@@ -1,94 +1,68 @@
-# Ubuntu 20.04 with nvidia-docker2 beta opengl support
-FROM osrf/ros:galactic-desktop
-USER root
+# Ubuntu 22.04 with ros2 desktop
+ARG ROS_DISTRO=humble
 
-ARG DEBIAN_FRONTEND=noninteractive
+FROM osrf/ros:${ROS_DISTRO}-desktop
 
-# Tools useful during development
-RUN apt-get update -qq \
-    && apt-get install --no-install-recommends -y -qq \
-    build-essential \
-    atop \
-    cmake \
-    cppcheck \
-    expect \
-    gdb \
-    git \
-    gnutls-bin \
-    libbluetooth-dev \
-    libccd-dev \
-    libcwiid-dev \
-    libfcl-dev \
-    libgoogle-glog-dev \
-    libspnav-dev \
-    libusb-dev \
-    python3-dbg \
-    python3-empy \
-    python3-numpy \
-    python3-setuptools \
-    python3-pip \
-    python3-venv \
-    software-properties-common \
-    vim \
-    net-tools \
-    iputils-ping \
-    xvfb \
-    curl 
+#create an user with a /home/cvar directory
```

Comment:
I dont quite like this, I would use /root/ for the docker as it simplifies some interactions with the container